tests/foreman/ui/test_computeresource_openstack.py

```diff
@@ -0,0 +1,362 @@
+
+        @setup: openstack hostname and credentials.
+        1. The addition of foreman-proxy-dns-reverse subnets.
+        2. The addition of foreman-proxy-dns-zone domain.
+        3. Addition of custom provisioning template.
+        to add ssh-key parameter.
+
+        @steps:
+        1. Population of images into satellite from openstack images.
+        2. Activation key and CV for the host.
+        3. Edit the required Operating system to
+        use the custom provisioning template.
+        4. Go to "Hosts --> New host".
+        5. Fill in the required details.(eg name,loc, org)(hostgroup)
+        6. Select openstack compute resource from "Deploy on" drop down.
+        7. Associate appropriate feature capsules.
+        8. Go to "operating system tab".
+        9. Select the appropriate image .
+        10. Associate the activation key and submit.
+
+        @Assert: The host should be provisioned successfully
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_openstack_with_compute_profile(self):
+        """ Provision a host on openstack compute resource with compute profile
+        default (3-Large)
+
+        @id: b1e1ad3e-9ad2-45c6-b8dd-4f39ce7729ca
+
+        @setup: openstack hostname ,credentials and provisioning setup.
+
+        @steps:
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select openstack compute resource from "Deploy on" drop down.
+        4. Select the "Compute profile" from the drop down.
+        5. Provision the host using the compute profile.
+
+        @Assert: The host should be provisioned successfully
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_openstack_with_custom_compute_settings(self):
+        """ Provision a host on openstack compute resource with
+         custom flavour and disk.
+
+        @id: 52c53d90-8eaa-47dd-8c44-6c03293cf64c
+
+        @setup: openstack hostname ,credentials and provisioning setup.
+
+        @steps:
+        1. Go to "Hosts --> New host".
+        2. Fill in the required details.(eg name,loc, org).
+        3. Select openstack custom compute resource from "Deploy on" drop down.
+        4. Select the custom compute profile" with custom disk size and flavor.
+        5. Provision the host using the compute profile.
+
+        @Assert: The host should be provisioned with custom settings
+
+        @caseautomation: notautomated
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_provision_openstack_with_host_group(self):
+        """ Provision a host on openstack compute resource with
+        the help of hostgroup.
+
+        @id: a3fecc66-0162-4b92-8341-14d255adc118
+
+        @setup: openstack hostname ,credentials provisioning
+         setup and hostgroup
```

Comment:
@setup: openstack hostname ,credentials provisioning
         setup and hostgroup